src/ws/cockpit.service.in

```diff
@@ -4,8 +4,14 @@ Documentation=man:cockpit-ws(8)
 Requires=cockpit.socket
 
 [Service]
+# Use this in the future once all our supported OSes support it
+#RuntimeDirectory=cockpit/tls
```

Comment:
It's a bit ugly to have this comment in a installed file, no?  Could we determine at configure time whether RuntimeDirectory works and just use it if so?

Reply:
Actually, in the meantime we dropped RHEL 7 support, so it may just work now. :-) I'll try that again. That still leaves the
```
# systemd ≥ 241 sets this automatically
Environment=RUNTIME_DIRECTORY=/run/cockpit/tls
```
(RHEL 8.{0,1} has systemd 239), but I think that's useful for admins when they look at the file and wonder why this is set explicitly.